packages/shared_preferences/shared_preferences/extension/devtools/config.yaml

```diff
@@ -0,0 +1,4 @@
+name: shared_preferences_tools
```

Comment:
let's rename to 'shared_preferences'. This should be the name of the package that is providing the extension: https://github.com/flutter/devtools/tree/master/packages/devtools_extensions#step-2-configure-your-extension